application/common/models/Method.php

```diff
@@ -131,7 +140,7 @@ public static function createAndSendVerification($userId, $type, $value)
                 throw new BadRequestHttpException($e->getMessage(), $e->getCode());
             }
         } else {
-            $method->value = $value;
+            $method->value = strtolower($value);
```

Comment:
Should this be `mb_strtolower(...)`? I'm not sure what all `$value` can represent here. (If we're using that string library, it probably also has some to-lower sort of function).